src/Fixer/Operator/TernaryOperatorSpacesFixer.php

```diff
@@ -114,6 +118,28 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         }
     }
 
+    /**
+     * @param int $index
+     *
+     * @return bool
+     */
+    private function belongsToGoToLabel(Tokens $tokens, $index)
+    {
+        if (!$tokens[$index]->equals(':')) {
+            return false;
+        }
+
+        $prevMeaningfulTokenIndex = $tokens->getPrevMeaningfulToken($index);
+
+        if (!$tokens[$prevMeaningfulTokenIndex]->isGivenKind(T_STRING)) {
+            return false;
+        }
+
+        $prevMeaningfulTokenIndex = $tokens->getPrevMeaningfulToken($prevMeaningfulTokenIndex);
+
+        return $tokens[$prevMeaningfulTokenIndex]->equalsAny([';', '{', '}', [T_OPEN_TAG], [T_OPEN_TAG_WITH_ECHO]]);
```

Comment:
I can imagine all the tokens, but `T_OPEN_TAG_WITH_ECHO`? @SpacePossum can you give an example with it, likely with test case as it would be not an obvious one.

Reply:
Thanks for the review!
I couldn't find an example so I removed the type. For the other types I've added tests.